tests/basic_integration/lib/clean_folders.sh

```
@@ -0,0 +1,14 @@
+clean_folders() {
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)